ffi/capi/bindings/cpp/icu4x/DecomposingNormalizer.d.hpp

```diff
@@ -89,7 +89,7 @@ class DecomposingNormalizer {
    *
    * See the [Rust documentation for `is_normalized_utf16`](https://docs.rs/icu/latest/icu/normalizer/struct.DecomposingNormalizerBorrowed.html#method.is_normalized_utf16) for more information.
    */
-  inline bool is_normalized_utf16(std::u16string_view s) const;
+  inline bool is_normalized16(std::u16string_view s) const;
```

Comment:
Question/Issue: Why did you change the function names? This seems like a breaking change

Reply:
`ComposingNormalizer` had the correct `diplomat:attr` regarding UTF-8/16 on its methods, while `DecomposingNormalizer` didn't. This fixes that.

This is a bindings-only change which I *hope* should be fine as a patch release before anyone is using 2.0.0 bindings.

Reply:
ok because you are following precedent from ComposingNormalizer